### DISCUSSION TITLE
Multiple organizations ?

### DISCUSSION THREAD

**Discussion by abdalem** [Infisical/infisical#479] [General]
HI ! Is there an plan to add the ability to handle multiple organizations ? We have a lot of customers that potentially have multiple projects and we do not want to have a large list of projects to scroll on our unique tenant, so we would like to split the projects by organizations. I have tried, on self-host plan, to add an organization to the mongo db but without success. It throws errors and never change as the user is owner of multiple organizations.

**Comment by dangtony98** [comment]
Hey @abdalem definitely — there's an open issue for that [here](https://github.com/Infisical/infisical/issues/414) actually that's being developed at the moment 😄

**Reply by mutschler** [reply]
so, any news on this? 
The issue was marked completed in Okt/Nov 2023 but i'm still unable to find any info neither in our instance nor in the docs.

same for:
- https://github.com/Infisical/infisical/pull/2225

**Reply by baek-sang** [reply]
Is this a paid plan?
